compiler-plugin/testData/api/annotationWithDefault.kt

```diff
@@ -146,7 +146,7 @@ public @interface JavaAnnotation2 {
 
 // FILE: JavaAnnotated.java
 
-@KotlinAnnotation(ConstKt.DebugKt)
```

Comment:
Can you keep this as is? This is probably caused during your rebase with latest main branch.